src/libraries/System.Text.Json/tests/Common/JsonTestHelper.cs

```diff
@@ -20,6 +20,16 @@ internal static partial class JsonTestHelper
         public const string SingleFormatString = "G9";
 #endif
 
+#if NETCOREAPP
+        public static Half NextHalf(Random random)
+        {
+            double mantissa = (random.NextDouble() * 2.0) - 1.0;
```

Comment:
Nit: are you looking to avoid creating values like `NaN` and infinity here? If not just generating a random pair of bytes and passing them to BitConverter.ToHalf might be simpler.

Reply:
Nevermind, I just realised you're following existing convention for other types.

Reply:
> are you looking to avoid creating values like NaN and infinity here?

Yes, that was the reason why I did this method after trying with BitConverter and the random pair of bytes.